### PR TITLE
feat: add bridge fee indexer and  cli

### PR DESCRIPTION
**Pull request by mkxbl**
1. index bridge fee withdraw tx event
2. query and withdraw bridge fee cli

> features have been tested

### CODE REVIEW

File: offchain-modules/packages/x/src/xchain/eth/contract.ts [hunk 1]
```diff
@@ -168,6 +169,21 @@ export class EthChain {
     return this.bridge.unlock(params, nonce, signatures);
   }
 
+  public async getUnlockMessageToSign(records: EthUnlockRecord[]): Promise<string> {
+    const bridge = new ethers.Contract(this.bridgeContractAddr, abi, this.provider);
+    const domainSeparator = await bridge.DOMAIN_SEPARATOR();
+    const typeHash = await bridge.UNLOCK_TYPEHASH();
+    const nonce: BigNumber = await bridge.latestUnlockNonce_();
+    console.log('nonce: ', nonce.toString());
```
Comment (solargatsby) [inline offchain-modules/packages/x/src/xchain/eth/contract.ts:177]: using logger.info

File: offchain-modules/packages/x/src/xchain/eth/contract.ts [hunk 2]
```diff
@@ -168,6 +169,21 @@ export class EthChain {
...
+    return buildSigRawData(domainSeparator, typeHash, records, nonce);
+  }
+
+  public async sendWithdrawBridgeFeeTx(records: EthUnlockRecord[], signatures: string[]): Promise<any> {
+    const nonce: BigNumber = await this.bridge.latestUnlockNonce_();
+    console.log('nonce: ', nonce.toString());
```
Comment (solargatsby) [inline offchain-modules/packages/x/src/xchain/eth/contract.ts:183]: using logger.info

File: offchain-modules/packages/app-cli/src/bridgeFee.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+      token: asset,
+      amount: ethAsset.parseAmount(amount[i]),
+      recipient: r,
+    };
+  });
+  const ethChain = new EthChain('verifier');
+  const message = await ethChain.getUnlockMessageToSign(withdrawRecords);
+  const privKeyPath = ForceBridgeCore.config.eth.multiSignKeys[0].privKey;
+  const privKey = parsePrivateKey(privKeyPath);
+  const { v, r, s } = ecsign(Buffer.from(message.slice(2), 'hex'), Buffer.from(privKey.slice(2), 'hex'));
```
Comment (solargatsby) [inline offchain-modules/packages/app-cli/src/bridgeFee.ts:151]: Don't need multi-sig?

File: offchain-modules/packages/x/src/xchain/eth/contract.ts (hunk 1 above)
Comment (mkxbl) [reply offchain-modules/packages/x/src/xchain/eth/contract.ts]: fixed

File: offchain-modules/packages/x/src/xchain/eth/contract.ts (hunk 2 above)
Comment (mkxbl) [reply offchain-modules/packages/x/src/xchain/eth/contract.ts]: fixed

File: offchain-modules/packages/app-cli/src/bridgeFee.ts (hunk 3 above)
Comment (mkxbl) [reply offchain-modules/packages/app-cli/src/bridgeFee.ts:151]: Every verifier use `forcecli fee generate-withdraw-tx-signature` cli to generate signature, then collector can ask verifiers for signatures and send withdraw-bridge-fee-tx with these signatures (`forcecli fee send-withdraw-tx -s [signatures] ...`).

**Pull request merged**